tests/lca/imagebasedinstall/mgmt/deploy/tests/e2e-deploy-test.go

```diff
@@ -0,0 +1,141 @@
+package deploy_test
+
+import (
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/bmh"
+	"github.com/openshift-kni/eco-goinfra/pkg/hive"
+	"github.com/openshift-kni/eco-goinfra/pkg/ibi"
+	"github.com/openshift-kni/eco-goinfra/pkg/namespace"
+	hiveV1 "github.com/openshift-kni/eco-goinfra/pkg/schemes/hive/api/v1"
+	"github.com/openshift-kni/eco-goinfra/pkg/schemes/hive/api/v1/none"
+	ibiv1alpha1 "github.com/openshift-kni/eco-goinfra/pkg/schemes/imagebasedinstall/api/hiveextensions/v1alpha1"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/secret"
+	"github.com/openshift-kni/eco-gotests/tests/lca/imagebasedinstall/mgmt/deploy/internal/tsparams"
+	. "github.com/openshift-kni/eco-gotests/tests/lca/imagebasedinstall/mgmt/internal/mgmtinittools"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+var _ = Describe(
+	"Performing image based installation",
+	Ordered,
+	Label(tsparams.LabelEndToEndDeployment), func() {
+		var (
+			ibiImageSetName string
+		)
+		BeforeAll(func() {
+			if MGMTConfig.Cluster == nil {
+				Skip("Failed to collect cluster info")
+			}
+
+			if MGMTConfig.SeedClusterInfo == nil || MGMTConfig.SeedClusterInfo.SeedClusterOCPVersion == "" {
+				Skip("Seed clusterinfo not supplied")
+			}
+
+			splitOCPVersion := strings.Split(MGMTConfig.SeedClusterInfo.SeedClusterOCPVersion, ".")
+
+			if len(splitOCPVersion) < 2 {
+				Skip("Could not determine short OCP version from seed cluster ocp version")
+			}
+
+			ibiImageSetName = strings.Join(splitOCPVersion[:2], ".")
+		})
+
+		It("is successful in connected environment", func() {
```

Comment:
what "is successful" mean here?

Reply:
"Performing image based installation is successful" - I'm going to update it to something more specific though